component/src/main/ballerina/redis/redis_client_endpoint.bal

```diff
@@ -0,0 +1,69 @@
+package ballerina.redis;
```

Comment:
Shall we add license header?

Reply:
Added in https://github.com/wso2-ballerina/package-redis/pull/9/commits/8bfc4913a7e31f674f955add1570b981063422ec